packages/@ama-sdk/showcase-sdk/openapitools.json

```diff
@@ -3,6 +3,14 @@
   "spaces": 2,
   "generator-cli": {
     "version": "6.6.0",
-    "storageDir": ".openapi-generator"
+    "storageDir": ".openapi-generator",
```

Comment:
We should also modify the package.json to call the generator with `--generator-key`